Tests/Harness/AppDelegate.swift

```diff
@@ -4,7 +4,7 @@ import UIKit
 class AppDelegate: UIResponder, UIApplicationDelegate {
   var window: UIWindow?
 
-  func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+  private func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
```

Comment:
Should this `private` be here? Given that it's meant to implement a `UIApplicationDelegate` method, I'd expect it would need the same visibility as the class itself.

Reply:
Ah yeah, missed that Xcode automatically added that `private` keyword. Removing.